commnad_line_rock_paper_scissors.py

```diff
@@ -6,6 +6,27 @@
 from rock_paper_scissors.game import Game
 
 
+def human_input_to_move(game, human):
+    choices = ', or '.join(
+        [
+            ', '.join(game.valid_choices[:-1]),
+            game.valid_choices[-1]
+        ]
+    )
+
+    try:
+        human.choice = game.convert_input_to_move(input('Choose {}: '.format(choices)))
```

Comment:
This function is quite lovely. To keep things more similar in `main`, I would actually keep the `human.choice` assignment in the same place (one line above) the corresponding assignment for `computer.choice`. This would entail a very small tweak here here:
```python
try:
    return game.convert_input_to_move(input('Choose {}: '.format(choices)))
except Exception:
```
(Note this also means that you'd remove the `return` statement that currently exists on line 23)

And another small change in `main`:
```python
while not game.is_complete:
    human.choice = human_input_to_move(game, human)
    computer.choice = computer_input_to_move(game)
```